src/Compilers/Core/Portable/Text/TextSpan.cs

```diff
@@ -204,12 +204,12 @@ public static TextSpan FromBounds(int start, int end)
         {
             if (start < 0)
             {
-                throw new ArgumentOutOfRangeException("start", CodeAnalysisResources.StartMustNotBeNegative);
+                throw new ArgumentOutOfRangeException(nameof(start), CodeAnalysisResources.StartMustNotBeNegative);
             }
 
             if (end < start)
             {
-                throw new ArgumentException("end", CodeAnalysisResources.EndMustNotBeLessThanStart);
+                throw new ArgumentException(CodeAnalysisResources.EndMustNotBeLessThanStart, nameof(end));
             }
```

Comment:
Order of parameters was wrong. Should we throw `ArgumentOutOfRangeException` instead?